src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -540,12 +540,7 @@ private void onPinnedEndpointChangedEvent(
                             + " {pinnedId}."));
         }
 
-        List<String> newPinnedIDList = Collections.EMPTY_LIST;
-        if (newPinnedEndpointID == null || "".equals(newPinnedEndpointID))
-        {
-            newPinnedIDList = Collections.singletonList(newPinnedEndpointID);
-        }
-
+        List<String> newPinnedIDList = (newPinnedEndpointID == null) ? Collections.EMPTY_LIST : Collections.singletonList(newPinnedEndpointID);
```

Comment:
This was previously only sending a Collections.EMPTY_LIST or a singletonList with a null or empty value